modules/apps/marketplace/marketplace-settings-web/src/main/resources/META-INF/resources/js/steps/Connect.tsx

```diff
@@ -0,0 +1,153 @@
+/* eslint-disable @liferay/empty-line-between-elements */
+
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayAlert from '@clayui/alert';
+import ClayButton from '@clayui/button';
+import {createResourceURL, fetch, objectToFormData, sub} from 'frontend-js-web';
+import pkceChallenge from 'pkce-challenge';
+import React from 'react';
+
+import Container from '../components/Container';
+import {Authorization, MarketplaceSettingsProps} from '../types';
+
+type ConnectProps = {
+	authorization: Authorization;
+	marketplaceSettingsProps: MarketplaceSettingsProps;
+	onDisconnect: () => void;
+	onNext: () => void;
+	setAuthorization: React.Dispatch<React.SetStateAction<Authorization>>;
+};
+
+const portletNamespace =
+	'_com_liferay_configuration_admin_web_portlet_InstanceSettingsPortlet_';
```

Comment:
Can't we pass this as a parameter in the function?

Reply:
It is possible to retrieve this value from the props inside the main component (index.tsx)